src/data/models/index.js

```diff
@@ -222,11 +222,16 @@ WeeklyBest.belongsTo(WeeklyWRs, {
   as: 'WeeklyWR',
 });
 
-Ranking.belongsTo(Kuski, {
+Ranking.hasOne(Kuski, {
```

Comment:
Can somebody confirm that there is in fact one ranking per kuski and that this should not have any negative effect on existing ranking functionality? I might try testing some ranking endpoints just to ensure there are no errors but i know nothing about the ranking system in general.

Reply:
Aren't these the exact opposite? As in that you are inversing the relationship? Since ranking table has KuskiIindex, that sort if implies that Ranking.belongsTo -> Kuski, not the othe way around?

Reply:
Not sure exactly what you mean. Could not inner join with sequelize unless I specified the "as" key in hasOne, forcing me to write the relation both ways which makes no sense imo.

Reply:
U know what I prolly have to adjust because ranking index col is not in the kuski table.  So I guess I'll go back to screwing around with sequelize again.